examples/pytorch/mnist2/README.md

```diff
@@ -0,0 +1,5 @@
+# PyTorch MNIST Image Classification Example
```

Comment:
Add a link to the original source: https://github.com/pytorch/examples/blob/main/mnist/main.py

Reply:
Added below.